docs/develop/api/networks.mdx

```diff
@@ -63,6 +63,10 @@ If you are searching for the protobuf interfaces, head over [here](https://buf.b
 | `https://rpc-evmos.mms.team:443`              | `Tendermint` `RPC`     | [MMS](https://mms.team/) | Pruned  |
 | `https://jsonrpc-evmos.mms.team:443`          | `Ethereum` `JSON-RPC`  | [MMS](https://mms.team/) | Pruned  |
 | `https://web3endpoints.com/evmos-mainnet`     | `Ethereum` `JSON-RPC`  | [BlockSpaces](https://www.blockspaces.com/web3-infrastructure) | Pruned |
+| `https://evmos-rest.antrixy.org`              | `Cosmos` `REST`        | Antrix Validators                               | Pruned  |
+| `https://evmos-grpc.antrixy.org`              | `Cosmos` `gRPC`        | Antrix Validators                               | Pruned  |
+| `https://evmos-rpc.antrixy.org`               | `Tendermint` `RPC`     | Antrix Validators                               | Pruned  |
+| `https://evmos-json.antrixy.org`              | `Ethereum` `JSON-RPC`  | Antrix Validators                               | Pruned  |
```

Comment:
These seem to be non-functional 🤔

Reply:
It is functional. Please check again.

Reply:
They do now 👍 they didn't work when I tried them back on Nov. 2nd